src/hotspot/os_cpu/linux_riscv/atomic_linux_riscv.hpp

```diff
@@ -130,8 +128,6 @@ template<>
 template<typename I, typename D>
 inline D Atomic::PlatformAdd<8>::add_and_fetch(I add_value, D volatile* dest,
                                                atomic_memory_order order) const {
-  fprintf(stderr, "Atomic::PlatformAdd<8>::add_and_fetch has been reached. Go to %s and check if it works\n", __FILE__);
```

Comment:
It seems like the function now can be seen in action

Reply:
Before deleting this line I tested the function by logging its input and return values. So it seems to work correctly